build.gradle

```diff
@@ -184,7 +184,7 @@ ext.externalDependency = [
   "lombok":"org.projectlombok:lombok:1.16.8",
   "mockRunnerJdbc":"com.mockrunner:mockrunner-jdbc:1.0.8",
   "xerces":"xerces:xercesImpl:2.11.0",
-  "typesafeConfig": "com.typesafe:config:1.3.0",
+  "typesafeConfig": "com.typesafe:config:1.2.1",
```

Comment:
I upgraded it to 1.3.0 in a PR earlier today, turned out 1.3.0 only supports JAVA 8